addons/storysource/src/StoryPanel.tsx

```diff
@@ -6,9 +6,11 @@ import {
   SyntaxHighlighter,
   SyntaxHighlighterProps,
   SyntaxHighlighterRendererProps,
-  createSyntaxHighlighterElement,
 } from '@storybook/components';
 
+// @ts-ignore
```

Comment:
I find it more useful with a `@ts-expect-error` comment that also explains why the error is expected

Reply:
Done